src/test/java/io/stargate/sgv2/jsonapi/api/model/command/deserializers/SortClauseDeserializerTest.java

```diff
@@ -131,6 +131,50 @@ public void vectorSearchInvalidSortClause() {
           .contains("Vector search can't be used with other sort clause");
     }
 
+    @Test
+    public void happyPathVectorizeSearch() throws Exception {
+      String json = """
+        {
+         "$vectorize" : "test data"
+        }
+        """;
+
+      SortClause sortClause = objectMapper.readValue(json, SortClause.class);
+
+      assertThat(sortClause).isNotNull();
+      assertThat(sortClause.sortExpressions()).hasSize(1);
+      assertThat(sortClause.sortExpressions().get(0).path()).isEqualTo("$vectorize");
+      assertThat(sortClause.sortExpressions().get(0).vectorize()).isEqualTo("test data");
+    }
+
+    @Test
+    public void vectorizeSearchNonText() {
+      String json = """
+        {
+         "$vectorize" : 0.55
+        }
+        """;
+
+      Throwable throwable = catchThrowable(() -> objectMapper.readValue(json, SortClause.class));
+
+      assertThat(throwable).isInstanceOf(JsonApiException.class);
+      assertThat(throwable.getMessage()).contains("$vectorize search needs to be text value");
+    }
+
+    @Test
+    public void vectorizeSearchObject() {
+      String json = """
+        {
+         "$vectorize" : {}
+        }
+        """;
+
+      Throwable throwable = catchThrowable(() -> objectMapper.readValue(json, SortClause.class));
+
+      assertThat(throwable).isInstanceOf(JsonApiException.class);
+      assertThat(throwable.getMessage()).contains("$vectorize search needs to be text value");
+    }
```

Comment:
Should also have failing test for case where there is non-vectorize sort clause in addition to vectorize one (assuming verification is done) -- unless it'd be part of one of follow-up issues.

Reply:
Added additional test case